.rubocop.yml

```diff
@@ -78,40 +83,25 @@ Style/ClassAndModuleChildren:
 Metrics/ClassLength:
   Enabled: false
 
-Layout/IndentationWidth:
-  Enabled: false
-
 Layout/EndAlignment:
-  Enabled: false
-
-Layout/ElseAlignment:
-  Enabled: false
-
-Performance/ChainArrayAllocation:
-  Enabled: true
+  EnforcedStyleAlignWith: start_of_line
 
 Style/BlockDelimiters:
   Enabled: false
 
 Style/RedundantSelf:
   Enabled: false
 
-Style/GuardClause:
-  Enabled: false
-
 Style/SymbolArray:
   Enabled: false
 
 Style/WordArray:
   Enabled: false
 
-Style/ParallelAssignment:
-  Enabled: false
-
-Style/PreferredHashMethods:
-  EnforcedStyle: verbose
+Style/RaiseArgs:
+  EnforcedStyle: compact
 
-Style/RedundantException:
+Style/ParallelAssignment:
   Enabled: false
```

Comment:
Since we had this cop in opposite, we have many `raise RuntimeError, ...` in the api, if you want to switch this cop, we need to clean them in this PR.

Reply:
`raise RuntimeError, ...` instead of `raise RuntimeError,.new` is just convention, but if we follow this convention all our raise are not homogeous

Reply:
effectively this is the case, I misunderstood this cop apparently! I like the new setting better because of homogeneity too!